src/xmipp/libraries/tomo/tomo_extract_subtomograms.cpp

```diff
@@ -165,18 +138,81 @@ void ProgTomoExtractSubtomograms::extractSubtomoFixedSize(MultidimArray<double>
 
 	if (downsampleFactor > 1)
 	{
-		downsample(fftSubtomoExtraction, fftSubtomo);
+		for (size_t k = 0; k < ZSIZE(fftSubtomo)/2; k++)
```

Comment:
Try not using more than 3 intentation levels. The previous version was OK and fixed by Fede.